source/store/AppContext.tsx

```diff
@@ -3,13 +3,18 @@ import React, { useState, useEffect } from "react";
 import env from "react-native-config";
 import StorageService, { APP_ENV_KEY } from "../services/StorageService";
 
-const AppContext = React.createContext({});
 interface Provider {
   mode: string;
   handleSetMode(newMode: string): void;
   isDevMode: boolean;
 }
 
+const AppContext = React.createContext<Provider>({
+  mode: "",
+  handleSetMode(newMode) {},
```

Comment:
This doesn't seem valid? Should be `handleSetMode: (newMode) => void` or something right?